### PR TITLE
Added: substr() method in javascript

### PR DESCRIPTION
**Pull request by Chandanmali**
Added substr method in javascript

### CODE REVIEW

File: en/strings/charAt.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+	// Finding the character at given index
+	let value = str.charAt(9);
+	console.log(value);
+}
+func();
+
+
+//Output
+t
```
Comment (sumn2u) [inline en/strings/charAt.md:50]: Are we missing the closing backticks of JavaScript syntax here?

File: en/strings/substr.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	let sub_str = str.substr(5, 6);
+	console.log(sub_str);
+}
+
+func();
+
+
+
+//Output
+a gre
```
Comment (sumn2u) [inline en/strings/substr.md:63]: Same goes here?

Comment (Chandanmali) [reply en/strings/substr.md:63]: @sumn2u  okk got your point 
check please

File: en/strings/charAt.md (hunk 1 above)
Comment (Chandanmali) [reply en/strings/charAt.md:50]: @sumn2u No, After function model we just call the function after the closing bracket.
This is a right code where we are call function after the bracket.

Comment (sumn2u) [reply en/strings/charAt.md:50]: Sorry for the confusion. I was asking if its possible to close the syntax of javascript?

(```javascript)

Comment (Chandanmali) [reply en/strings/charAt.md:50]: @sumn2u okk! got your point, Updated
Can you check now?

**Review by sumn2u — APPROVED** [review]
LGTM

**Pull request merged**